x-pack/auditbeat/module/system/sockets/_meta/fields.yml

```diff
@@ -0,0 +1,66 @@
+- name: sockets
+  type: group
+  description: >
+    `sockets` contains socket information.
+  release: experimental
+  fields:
+  - name: socket
+    type: array
+    description: >
+        Information about one or more sockets.
+    fields:
+    - name: family
```

Comment:
We've beefed up the ability to specify network protocol details just before ECS 1 beta1. You'll want to review the [network field set](https://github.com/elastic/ecs#network) again and potentially leverage some of it.

It's now able to track multiple layers of details around protocols:

- `.type` for IPv4/6, ICMP
- `.iana_number` (optional) and `.transport` for TCP/UDP, etc
- `.protocol` for the application layer, such as HTTP
- `.direction` for inbound or outbound
- and eventually perhaps `.[inbound|outbound].[bytes|packets]`, if we end up tracking to this level of details

Reply:
Changed to `network.type` and `network.direction`.